climpred/tests/test_logging_hindcast.py

```diff
@@ -1,172 +0,0 @@
-def test_logg_compute_hindcast_alignment(
-    hind_ds_initialized_1d, reconstruction_ds_1d, caplog, alignment
-):
-    """
-    Checks that logging in compute hindcast captures the correct verification-time
-    matching based on `alignment`.
-    """
-    # setting up data for cftime init/time
-    hind_ds_initialized_1d['init'] = xr.cftime_range(
-        start=str(hind_ds_initialized_1d.init.min().values),
-        freq='YS',
-        periods=hind_ds_initialized_1d.init.size,
-    )
-    hind_ds_initialized_1d.lead.attrs['units'] = 'years'
-    reconstruction_ds_1d['time'] = xr.cftime_range(
-        start=str(reconstruction_ds_1d.time.min().values),
-        freq='YS',
-        periods=reconstruction_ds_1d.time.size,
-    )
-
-    with caplog.at_level(logging.INFO):
-        compute_hindcast(hind_ds_initialized_1d, reconstruction_ds_1d)
-        # check for each record
-        for i, record in enumerate(caplog.record_tuples):
-            print(record)
-            lead = hind_ds_initialized_1d.isel(lead=i).lead.values.astype('int')
-            assert f'at lead={str(lead).zfill(2)}' in record[2]
-            if alignment == 'maximize':
-                # check that
-                first_year_per_lead = (
-                    hind_ds_initialized_1d.init.min().dt.year.values + lead
-                )
-                assert f'{first_year_per_lead}' in record[2]
-            elif alignment == 'same_verif':
-                # check that verification time is always the same
-                # const_verif_length = xx
-                pass  # skip test for now
-                # get verification times
-                # first = 1980  # made up
-                # last = 2000  # made up
-                # assert f'{first}' in record[2] and f'{last}' in record[2]
-                # check always same time length verified
-                # assert const_verif_length == last_year_per_lead - first_year_per_lead
-            elif alignment == 'init':
-                # check that verification length is always the same and init stay the
-                # same, therefore verification time should monotonically decrease for
-                # monotonic lead increases
-                # const_verif_length = xx
-                first_year_per_lead = (
-                    hind_ds_initialized_1d.init.min().dt.year.values + lead
-                )
-                assert f'{first_year_per_lead}' in record[2]
-                last_year_per_lead = (
-                    hind_ds_initialized_1d.init.max().dt.year.values
-                    + lead
-                    - hind_ds_initialized_1d.lead.size
-                )
-                assert f'{last_year_per_lead}' in record[2]
-                # check always same time length verified
-                # assert const_verif_length == last_year_per_lead - first_year_per_lead
-
-
-@pytest.mark.parametrize('alignment', ['init', 'same_verif', 'maximize'])
```

Comment:
do you test logging for maximize somewhere?

Reply:
No, maximize isn't in this PR. I'm going to follow this with a PR for `same_verifs` and one for `maximize`. They'll follow similar structure to this one.